sklearn/ensemble/_forest.py

```diff
@@ -615,6 +615,12 @@ def feature_importances_(self):
     )
     @property
     def n_features_(self):
+        """ The number of features when ``fit`` is performed.
+        Returns
+        -------
+        n_features_in_ : int
+            The number of features when ``fit`` is performed.
+        """
```

Comment:
So the CI will probably fail but it will be fixed thanks to #20385 

However, could you make these small changes:

```suggestion
        """Number of features when fitting the estimator.

        Returns
        -------
        n_features_in_ : int
            The number of features when fitting the estimator.
        """
```